File: commands/kanye.py
#!/usr/bin/env python3
import requests


async def kanye(ctx):
    r = requests.get("https://api.kanye.rest/").json()

    quote = r["quote"]
    content = f"<a:kanyePls:1081048056058871808>  💬  ❝ {quote} ❞"

    await ctx.respond(content=content)

File: commands/kanye.ts
import { DiscordInteraction, InteractionResponse } from "../types/discord";
import { textResponse, errorResponse } from "../utils/discord";

// Animated Kanye emoji from MEGACORD
const KANYE_EMOJI = "<a:kanyePls:1081048056058871808>";

export async function handleKanye(
  _interaction: DiscordInteraction,
): Promise<InteractionResponse> {
  try {
    const response = await fetch("https://api.kanye.rest/");
    const data = (await response.json()) as { quote: string };

    return textResponse(`${KANYE_EMOJI}  💬  ❝ ${data.quote} ❞`);
  } catch (error) {
    console.error("Kanye API error:", error);
    return errorResponse("Failed to get a Kanye quote. Try again later.");
  }
}
